rolling-update-kubernetes/src/main/resources/reference.conf

```diff
@@ -34,6 +34,11 @@ akka.rollingupdate.kubernetes {
     pod-name = ""
     pod-name = ${?KUBERNETES_POD_NAME}
 
+    # Annotations to check to determine the revision.  The first one in the list which matches
+    # wins.  The default is suitable for "vanilla" Kubernetes Deployments, but other CI/CD systems
+    # may set a different annotation.
+    revision-annotation = [ "deployment.kubernetes.io/revision" ]
```

Comment:
In practice, will there actually be multiple configured? If yes, should this be plural, like `revision-annotations`?

Reply:
Probably not, but it's a somewhat bigger change down the road to go from exactly-one to multiple than to go from nothing to multiple.

Reply:
if the first matched one wins, why not just keep it as exactly one match and set it with 
revision-annotation = "deployment.kubernetes.io/revision" 
we can override it in our setting with other annotation

Reply:
I agree, skip the list, and make it a single value.